src/daal4py.cpp

```diff
@@ -378,6 +378,11 @@ daal::data_management::NumericTablePtr make_nt(PyObject * obj)
                         break;
                     }
 
+                    if (!array_is_behaved(ary))
+                    {
+                        std::cerr << "Can't get the data correctly without a copy.\n";
+                    }
+
```

Comment:
This check must either raise an exception (e.g. break) or print a warning *and* auto-convert. It's not a good idea to proceed if we know it will most likely produce a seg-fault.

Can't we do this in the generated cython code?

Reply:
The message could be more discriptive since we want to tell the user that he can work-around this (by making a contiguous copy).
Maybe "Cannot operate on column x because it is non-contiguous. Please make it contiguous before passing it to daal4py."
Or "Warning: implicitly created a copy of column x. Make it contiguous before passing it to daal4py to avoid potential negative performance implications."
If a warning and implicit conversion, the user needs to be able to switch the message of (not the conversion).